source/elements/oneMKL/source/domains/data_fitting/terms.rst

```diff
@@ -0,0 +1,324 @@
+.. _terms:
+
+Common Terms
+============
+
+.. contents::
+    :local:
+    :depth: 2
+
+Glossary
+--------
+
+Assume we need to interpolate a function ``f(x)`` on the ``[a, b)`` interval using splines.
+
+Let's break ``[a, b)`` into sub-intervals by ``n`` points :math:`x_i`
+called partition points or simply **partition**.
+**Function values** at these points (:math:`y_i = f(x_i), i=1,\cdots,n`) are also given.
+
+Spline has ``k`` degree if it can be expressed by the following polynomial:
+
+.. math::
+  P\left( x \right) =
+  c_{1} +
+  c_{2}\left( x - x_i \right) +
+  c_{3}{\left( x - x_i \right)}^2 + \cdots +
+  c_{k-1}{\left( x - x_i \right)}^k.
+
+Splines are constructed on :math:`[x_i, x_{i+1}), i=1,\cdots,n-1` sub-intervals.
+So, for each sub-interval there are following polynomials:
+
+.. math::
+  P_i\left( x \right) =
+  c_{1,i} +
+  c_{2,i}\left( x - x_i \right) +
+  c_{3,i}{\left( x - x_i \right)}^2 + \cdots +
+  c_{k-1,i}{\left( x - x_i \right)}^k.
+
+:math:`c_{j,i}, j=1,\cdots,k, i=1,\cdots,n` are called spline **coefficients**.
+
+Function is interpolated at points of [a, b).
```

Comment:
Given a interpolant spline of `:math:`f(x)` on a partition of :math:`[a,b)`, The interpolant function can be evaluated at any point in the interval :math:`[a,b)`.  Such points to evaluate the interpolant at will be referred to as **interpolation sites** or simply **sites**.  Sites can correspond to partition points, but are not required to be.  They must however be in the interval :math:`[a,b)` to be considered interpolation sites.


side question:  can they also be at the value "b" ?  so this could be :math:`[a,b]` ?

Reply:
Actually sites can be `< a` or `>= b`, but in this case first or last sub-interval will be used for interpolation.